source/h1_encoder.c

```diff
@@ -61,9 +63,20 @@ static int s_scan_outgoing_headers(
                     has_body_headers = true;
                 }
             } break;
-            case AWS_HTTP_HEADER_TRANSFER_ENCODING:
-                AWS_LOGF_ERROR(AWS_LS_HTTP_STREAM, "id=static: Sending of chunked messages not yet implemented");
-                return aws_raise_error(AWS_ERROR_UNIMPLEMENTED);
+            case AWS_HTTP_HEADER_TRANSFER_ENCODING: {
+                if (0 == header.value.len) {
+                    AWS_LOGF_ERROR(AWS_LS_HTTP_STREAM, "id=static: Transfer-Encoding must include a valid value");
+                    return aws_raise_error(AWS_ERROR_HTTP_INVALID_HEADER_VALUE);
+                }
+                struct aws_byte_cursor trimmed_value = aws_strutil_trim_http_whitespace(header.value);
+                struct aws_byte_cursor chunked = aws_byte_cursor_from_c_str("chunked");
```

Comment:
This can be predefined at static file scope.

Reply:
done.